- fix for when keyframes are combined.

kodybrown opened the pull request:
When I have created multiple keyframes (dynamically via sass) they are combined without semi-colon.

Apparently, the document parser trims trailing semi-colons from stylesheet rules..

kodybrown committed:
- fix for when keyframes are combined.

kodybrown committed:
- added proper copyright notice.

Prinzhorn commented:
Please post an example. Or even better add a test case to the test suit. Because just by looking at the code I don't know what input you're trying to handle.

kodybrown committed:
- Created three new tests, two of which are fixed.
  - Class attributes without trailing spaces
  - Stylesheet with multiple @-skrollr-keyframes and no trailing spaces
  * External stylesheet precedence

kodybrown committed:
- fixed typo?

kodybrown committed:
Merge remote-tracking branch 'upstream/master'

Conflicts:
	README.md
	src/skrollr.stylesheets.js
	test/tests.js

evolve2k commented:
Ping @kodybrown, please please post us an example so we can get this pull request wrapped up :gift:

kodybrown commented:
There are three tests in there?

evolve2k commented:
@Prinzhorn I had a look at the tests, looks ok to me. Lets get this merged.

Prinzhorn commented:
@evolve2k I've added you as a collaborator. Feel free to give some new live to this repo.

kodybrown committed:
Merge branch 'Prinzhorn:master' into master
